Visualizer/KIFEventVisualizer.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/Foundation.h>
+
+@interface KIFEventVisualizer : NSObject
+
++ (instancetype)sharedVisualizer;
```

Comment:
Do we want to set `- (instancetype) init` to `NS_UNAVAILABLE`? Or would anyone else be calling the `init` method?

Reply:
Good callout, since we are creating our own window overlay we probably want only 1 being created.

Reply:
Had to make an `initPrivate` method because since it's marked unavailable using `init` internally also breaks

Reply:
Looks good!